nservicebus/testing/index_content_testing_[6,).partial.md

```diff
@@ -18,14 +40,39 @@ The following unit test checks that the total amount has the discount applied:
 
 snippet: SagaTest
 
+
 ## Testing a behavior
 
+[Message pipeline behaviors](/nservicebus/pipeline/manipulate-with-behaviors.md) also can be tested, but using a different testable context objects, one for each of the [various pipeline stages](/nservicebus/pipeline/steps-stages-connectors.md) that can be extended with a behavior.
+
+| Behavior Context | Testable Implementation |
```

Comment:
as said on various issues: I'm not a fan of listing all behaviors and their testing counterparts as I think it's not worth the maintenance effort compared to just documenting the naming convention for the testable behaviors. No strong opinion on that though.